packages/shell-api/src/shell-bson.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint prefer-rest-params: 0, @typescript-eslint/ban-ts-ignore: 0 */
+import bson from 'bson';
+
+function dateHelper(...args: any[]): Date {
+  if (args.length === 0) {
+    return new Date();
+  }
+  if (args.length === 1) {
+    return new Date(args[0]);
+  }
+  // @ts-ignore
+  return new Date(Date.UTC(...args));
+}
+export default {
+  RegExp: RegExp,
+  DBRef: bson.DBRef,
+  DBPointer: bson.DBPointer,
+  Map: bson.Map,
+  MaxKey: bson.MaxKey,
+  MinKey: bson.MinKey,
+  ObjectId: bson.ObjectID,
+  BSONSymbol: bson.BSONSymbol,
+  Timestamp: bson.Timestamp,
+  Code: function(c, s): bson.Code {
+    return new bson.Code(c, s);
+  },
+  NumberDecimal: function(s): bson.Decimal128 {
+    if (s === undefined) {
+      s = '0';
+    }
+    return bson.Decimal128.fromString(s.toString());
+  },
+  NumberInt: function(s): any {
+    return parseInt(s, 10);
+  },
+  NumberLong: function(v): bson.Long {
+    if (v === undefined) {
+      v = 0;
+    }
+    return bson.Long.fromNumber(v);
+  },
+  Date: function(): Date|string {
```

Comment:
`arguments` is still there but is somehow deprecated: 
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Deprecated_and_obsolete_features
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

```
  Date: function(...args: DateConstructorArguments): Date|string {
    const date = dateHelper(...args);
```